client/src/core/client/admin/routes/Configure/sections/Moderation/PremoderateEmailAddressConfig.tsx

```diff
@@ -0,0 +1,69 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+import { graphql } from "react-relay";
+
+import {
+  FieldSet,
+  FormField,
+  FormFieldHeader,
+  Label,
+} from "coral-ui/components/v2";
+
+import ConfigBox from "../../ConfigBox";
+import Header from "../../Header";
+import HelperText from "../../HelperText";
+import OnOffField from "../../OnOffField";
+
+// eslint-disable-next-line no-unused-expressions
+graphql`
+  fragment PremoderateEmailAddressConfig_formValues on Settings {
+    premoderateEmailAddress {
+      tooManyPeriods {
+        enabled
+      }
+    }
+  }
+`;
+
+interface Props {
+  disabled: boolean;
+}
+
+const PremoderateEmailAddressConfig: FunctionComponent<Props> = ({
+  disabled,
+}) => {
+  return (
+    <ConfigBox
+      title={
+        <Localized id="configure-moderation-premoderateEmailAddress-title">
+          <Header container={<legend />}>Email address</Header>
+        </Localized>
+      }
+      container={<FieldSet />}
+    >
+      <FormField container={<FieldSet />}>
+        <FormFieldHeader>
+          <Localized id="configure-moderation-premoderateEmailAddress-enabled">
+            <Label component="legend">
+              Premoderate emails with too many periods
```

Comment:
nit: it might be nice to update for consistency between `premoderate` and `pre-moderate` in the text here? I think in other places we use `pre-moderate`

Reply:
fixed!